docker/docker-compose.base.yml

```diff
@@ -0,0 +1,75 @@
+# When increasing number of nodes in this file, change default value of --validators-count param in e2e-tests/src/config.rs
+
+services:
+  Node0:
+    extends:
+      file: common.yml
+      service: AlephBootNode
+    container_name: Node0
+    environment:
+      - RPC_PORT=9933
+      - WS_PORT=9943
+      - PORT=30333
+      - VALIDATOR_PORT=30343
+      - PUBLIC_VALIDATOR_ADDRESS=127.0.0.1:30343
+      - NAME=Node0
+
+  Node1:
+    extends:
+      file: common.yml
+      service: AlephNonBootNode
+    container_name: Node1
+    environment:
+      - RPC_PORT=9934
+      - WS_PORT=9944
+      - PORT=30334
+      - VALIDATOR_PORT=30344
+      - PUBLIC_VALIDATOR_ADDRESS=127.0.0.1:30344
+      - NAME=Node1
+      - BASE_PATH=/data/5GBNeWRhZc2jXu7D55rBimKYDk8PGk8itRYFTPfC8RJLKG5o
```

Comment:
just for clarity - please add a comment about seed (just like `Node0` has in `common.yml`)